clients/hadoop-to-venice-bridge/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -639,6 +643,11 @@ private PushJobSetting getPushJobSetting(String veniceControllerUrl, VenicePrope
     pushJobSettingToReturn.kafkaInputCombinerEnabled = props.getBoolean(KAFKA_INPUT_COMBINER_ENABLED, false);
     pushJobSettingToReturn.suppressEndOfPushMessage = props.getBoolean(SUPPRESS_END_OF_PUSH_MESSAGE, false);
     pushJobSettingToReturn.deferVersionSwap = props.getBoolean(DEFER_VERSION_SWAP, false);
+    pushJobSettingToReturn.repushTtlInHours = props.getLong(REPUSH_TTL_IN_HOURS_OVERRIDE, -1);
```

Comment:
Maybe define a constant for the default repush TTL value since it is used at several places?

Reply:
Good idea. I found we use `-1` a lot as a place holder for invalid input. I can possibly define such a constant